examples/bpi-monolith-test-suite/readme.md

```diff
@@ -0,0 +1,4 @@
+* install nodejs LTS
```

Comment:
Would help with links to where the LTS can be downloaded from, and which version of npm to use.

Reply:
Will be tackled as part of work on the readme from [here ](https://github.com/eea-oasis/baseline/projects/2)